#!/usr/bin/env python
PACKAGE = "iris_cobot"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("force_div",    double_t,      0, "Force conversion constant",     30,  0, 100)
gen.add("torque_div",   double_t,      0, "Torque conversion constant",    3,   0, 10)

gen.add("force_sensibility",  double_t,   0, "Activation sensibility of FT",   0.1, 0, 1)
gen.add("torque_sensibility",  double_t,   0, "Activation sensibility of FT",   0.1, 0, 1)

exit(gen.generate(PACKAGE, "iris_cobot", "FT_to_Vel"))